listen_test.go

```diff
@@ -445,6 +446,32 @@ func Test_Listen_ListenerNetwork(t *testing.T) {
 	}))
 
 	require.Contains(t, network, "0.0.0.0:")
+
+	go func() {
```

Comment:
This is only testing that the listener starts.

We need a test where a request a I sent to the server using a client